src/kbmod/search/image_kernels.cu

```diff
@@ -8,6 +8,8 @@
 #ifndef IMAGE_KERNELS_CU_
 #define IMAGE_KERNELS_CU_
 
+#define MAX_STAMP_IMAGES 140
```

Comment:
How was this number determined? What might happen if someone changes it?

Reply:
This was set to equal the number used in the main search algorithm. It balances the memory used on the GPU with the number of images supported. Increasing the number just uses more memory. Decreasing it could cause a problem, so I added an assert to check that it is valid in case someone reduces it too much.